docker/base.Dockerfile

```diff
@@ -127,13 +127,13 @@ COPY docker/utils.sh /tmp
 RUN \
 . /tmp/utils.sh && \
 groupadd -g 999 docker && \
-create_users "${BBS_USERS},bbs_user/1000" "docker" && \
+create_users "${BBS_USERS},guest/1000" "docker" && \
```

Comment:
There was a typo here anyway (creating user `bbs_user` but trying to use `bbsuser` further down) so while correcting it I thought `bbs` is not actually 100% correct any more since we're `bluesearch` and `bs_user` sounds rude, so I ended up just taking `guest`.